### DISCUSSION TITLE
Repo Latest, Stars and Forks not displaying

### DISCUSSION THREAD

**Discussion by 3goats** [squidfunk/mkdocs-material#3818] [General]
Hi - Am I missing something. 

I've specified my `repo_url: ` in the `MkDocs.yaml` config and the icon and working link to my repo is rendered. However, it's not displaying Latest, Stars and Forks info. Am I missing something here ?  Is the fact it's a private repo affecting things ?

**Comment by squidfunk** [comment]
Correct, stars and forks do not work for private repositories. You would need to perform an authenticated request, and since the information is retrieved in the browser of the user, you would need to expose your credentials to every user. Not a good idea 😉

**Comment by frederic-loui** [comment]
Hi, I have also specified  `repo_url`, but I observe a different issue and the repository is public. Apology for the dumb question but is a way to select which infmraton is displayed for a GitHub repository ? In the screenshot below GitHub repository `tag` is not displayed. It should be [v23.7.2](https://github.com/rare-freertr/freeRtr/tags) no ?
![Capture d'écran 2023-07-05 094728](https://github.com/squidfunk/mkdocs-material/assets/9639687/583b89ad-4840-4403-988f-18dee44adbca)

**Reply by squidfunk** [reply]
Tags are not supported. You need to create a GitHub release, [as mentioned in our documentation](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#repository). Related:

- https://github.com/squidfunk/mkdocs-material/issues/2943
- [Why tags are not supported](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#fn:1)
